index.js

```diff
@@ -3,10 +3,18 @@
 
 module.exports = {
   name: 'ember-memory-stats',
+  
+  isEnabled: function () {
+    var options = this.app.options.browseryStats;
+    return typeof options === 'undefined' || !!options;
```

Comment:
Kept it to todays behavior where it's enabled when not defined.. don't want to break anyone who updates.